flow360/component/simulation/framework/base_model.py

```diff
@@ -41,6 +42,12 @@ class Flow360BaseModel(pd.BaseModel):
 
     def __init__(self, filename: str = None, **kwargs):
         model_dict = self._handle_file(filename=filename, **kwargs)
+        keys_to_remove = []
+        for property_name in model_dict.keys():
+            if re.match(r"^_[^_]", property_name):
+                keys_to_remove.append(property_name)
+        for key in keys_to_remove:
+            model_dict.pop(key)
```

Comment:
this should be moved to a separate function `handle_private_fields()`. Left JIRA ticket for it.